examples/remote-manifests/README.md

```diff
@@ -0,0 +1,9 @@
+# remote-manifests
+
+This example shows how you to specify remote resources for a component's `manifests`.
+
+:::info
+
+To view the example source code, select the `Edit this page` link below the article and select the parent folder.
+
+:::
```

Comment:
I would like to see at least a short snippet of code on the example page (not the full zarf.yaml) but enough that people viewing it can get the gist without going to GitHub

Reply:
Also, I dont think this entire `Go to Github` flow works well at all, as even if we used kiwix it would not grab that, thereby breaking the ability to _air gap_ our docs.

Reply:
Oh yeah it is totally borked. If you make an issue I am good with that too - we really need to make examples more useful.